### PR TITLE
Modify create API output directory and remove redundant directories

### PR DESCRIPTION
**Pull request by ashera96**

### CODE REVIEW

File: impl/createApi.go [hunk 1]
```diff
@@ -157,9 +157,9 @@ func CreateAPI(filePath, namespace, serviceUrl, apiName, version string, isDryRu
 		return err
 	}
 
-	dirPath := path.Join(utils.GetAPKCTLHomeDir(), apiName)
+	dirPath := path.Join(utils.GetAPKCTLHomeDir(), utils.APIProjectsDir, apiName)
 
-	os.Mkdir(dirPath, os.ModePerm)
+	os.MkdirAll(dirPath, os.ModePerm)
```
Comment (BLasan) [inline impl/createApi.go:162]: Why `mkdirAll` ? Don't we have to specify the path correctly?

File: utils/utils.go [hunk 2]
```diff
@@ -19,13 +19,15 @@
...
 import (
+	"os"
 	"strings"
 
 	k8sUtils "github.com/BLasan/APKCTL-Demo/k8s"
 )
 
 func GetAPKCTLHomeDir() string {
-	return "/Users/benura/Desktop/APKCTL"
+	pwd, _ := os.Getwd()
```
Comment (BLasan) [inline utils/utils.go:29]: Shouldn't we handle the error returned here?

File: impl/createApi.go (hunk 1 above)
Comment (ashera96) [reply impl/createApi.go:162]: `Mkdir` won't work in this instance as we need nested directories to get created. The changes proposed through this PR need the following nested directory format: `target/apis/<api_name>/*`. As per the codebase prior to these changes, `Mkdir` was sufficient to generate the output we wanted.

`MkdirAll` creates a directory named path, along with any necessary parents, and returns nil, or else returns an error.

File: utils/utils.go (hunk 2 above)
Comment (ashera96) [reply utils/utils.go:29]: Will fix with the next PR

**Pull request merged**